dashboard/app/controllers/resources_controller.rb

```diff
@@ -18,6 +18,10 @@ def create
         include_in_pdf: resource_params[:include_in_pdf]
       }
     )
+    if resource_params[:courseVersionId]
+      course_version = CourseVersion.find(resource_params[:courseVersionId])
+      resource.course_version = course_version if course_version
+    end
```

Comment:
`find` will raise `ActiveRecord::RecordNotFound` if it doesn't find what it's looking for, so I would recommend either rescuing that error here, or else use a method like `find_by` or `find_by_id` which will return nil if not found.

If the course version is not found, it also seems like it would be better to return a 4xx error to the client rather than silently ignoring the course version id param.